libs/dkg/examples/dkg/main.cpp

```diff
@@ -148,7 +145,7 @@ int main()
     }
 
     // Need to increase this depending on number of nodes to complete 3 rounds of DRB
-    std::this_thread::sleep_for(std::chrono::seconds(25));
+    std::this_thread::sleep_for(std::chrono::seconds(600));
```

Comment:
Does your dkg_service have is_synced_ ? you could use that

Reply:
Yes, I didn't realise that had been added. Thanks.